tests/test_controller.py

```diff
@@ -32,12 +32,16 @@ def test_registry(controller, decentralizedController):
     with raises(TransactionFailed): decentralizedController.assertOnlySpecifiedCaller(tester.a2, key2, sender = tester.k0)
     with raises(TransactionFailed): controller.assertOnlySpecifiedCaller(tester.a2, key2, sender = tester.k2)
 
-def test_suicideFunds(controller, decentralizedController, controllerUser):
-    with raises(TransactionFailed): controller.suicideFunds(controllerUser.address, tester.a0, sender = tester.k2)
+def test_suicideFunds(contractsFixture, controller, decentralizedController, controllerUser):
+    legacyRepContract = contractsFixture.contracts['LegacyRepContract']
+    legacyRepContract.faucet(0)
+    legacyRepContract.transfer(controllerUser.address, 40 * 10**18)
+    with raises(TransactionFailed): controller.suicideFunds(controllerUser.address, tester.a0, [legacyRepContract.address], sender = tester.k2)
     assert decentralizedController.owner() == bytesToHexString(tester.a0)
-    with raises(TransactionFailed): decentralizedController.suicideFunds(controllerUser.address, tester.a0, sender = tester.k0)
-    assert controller.suicideFunds(controllerUser.address, tester.a0, sender = tester.k0)
+    with raises(TransactionFailed): decentralizedController.suicideFunds(controllerUser.address, tester.a0, [legacyRepContract.address], sender = tester.k0)
+    assert controller.suicideFunds(controllerUser.address, tester.a0, [legacyRepContract.address], sender = tester.k0)
```

Comment:
Recommend creating a new test like `test_suicide_tokens` that does the token stuff and asserts that the tokens were correctly suicided.  Such a test will fail now since a block of the code is commented out, but we can mark just that test as skipped and give the reason for the skip so that once we uncomment the code we just have to unmark it.

Reply:
How do you mark a test as skipped

Reply:
https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions

```
from pytest import mark
@mark.skip(reason="This test currently fails for unknown reasons, need to investigate before release.")
def test_stuff():
    assert False
```